assets/screens/input_maps.onj

```diff
@@ -1,37 +1,78 @@
-
 use Screen;
 
 defaultInputMap: [
     {
-        keycode: keys.F,
-        modifiers: [],
-        action: $ToggleFullscreenKeyAction {
-            width: 700,
-            height:(700*9)/16
-        }
+        priority: (2.0 pow 30.0)#int,
+        triggers: [
+            {
+                keycode: keys.F,
+                modifiers: [
+                ],
+                action: $ToggleFullscreenKeyAction {
+                    width: 700,
+                    height: (700 * 9) / 16
+                }
+            },
+        ],
+    },
+    { // block everything else, if you are in an inputfield (has no action, but a high priority)
+        priority: (2.0 pow 30.0)#int -1,
```

Comment:
yes, because i wanted to make sure that you are happy while debugging. Additionally, at a later point it won't be F, but F11, since every game has that, and you shouldn't be forced to be in fullscreen while typing something, because you might want to look something up or so